modules/virt-example-inherit-static-ip-from-nic.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+:_content-type: REFERENCE
+[id="virt-example-inherit-static-ip-from-nic_{context}"]
+= Example: Linux bridge interface node network configuration policy to inherit static IP address from the NIC attached to the bridge
+
+Create a Linux bridge interface on nodes in the cluster and transfer the static IP configuration of the NIC to the bridge by applying a single `NodeNetworkConfigurationPolicy` manifest to the cluster.
```

Comment:
I might start this with "You can create..." because it's a reference module. The imperative form here reminds me of a procedure.

Reply:
The other examples in the assembly use the imperative form, so I went with that wording here as well.